modules/configuring-quota-management-for-distributed-workloads.adoc

```diff
@@ -21,13 +21,28 @@ ifdef::cloud-service[]
 * You have downloaded and installed the OpenShift command-line interface (CLI). See link:https://docs.openshift.com/dedicated/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI] (Red Hat OpenShift Service on AWS).
 endif::[]
 
-ifndef::upstream[]
-* You have enabled the required distributed workloads components as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads#configuring-the-distributed-workloads-components_distributed-workloads[Configuring the distributed workloads components].
-endif::[]
+
 ifdef::upstream[]
-* You have enabled the required distributed workloads components as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-the-distributed-workloads-components_distributed-workloads[Configuring the distributed workloads components].
+* You have installed the required distributed workloads components as described in link:{odhdocshome}/installing-open-data-hub/#installing-distributed-workload-components_installv2[Installing the distributed workloads components].
+endif::[]
+
+ifdef::self-managed[]
+
+ifndef::disconnected[]
+* You have installed the required distributed workloads components as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-and-deploying-openshift-ai_install#installing-distributed-workload-components_component-install[Installing the distributed workloads components].
 endif::[]
 
+ifdef::disconnected[]
+* You have installed the required distributed workloads components as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/installing-and-deploying-openshift-ai_install#installing-distributed-workload-components_component-install[Installing the distributed workloads components].
+endif::[]
+
+endif::[]
```

Comment:
Because there is only one SM "Working with distributed workloads" book (there is not a separate one for disconnected), this could be:
* You have installed the required distributed workloads components as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-and-deploying-openshift-ai_install#installing-distributed-workload-components_component-install[Installing the distributed workloads components] or link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/installing-and-deploying-openshift-ai_install#installing-distributed-workload-components_component-install[Installing the distributed workloads components in a disconnected environment].

Reply:
The link has changed, the info is now in the installation guide, not in the DW guide.